components/molecules/Category/Box.js

```diff
@@ -0,0 +1,22 @@
+import HeadingH3 from "../../atoms/Heading/HeadingH3"; 
+import TextP from '../../atoms/Text/TextP'
+import Button from '../../atoms/Button/Button'
+import {Container,LeftSection,RightSection} from  './styles'
+import Image from 'next/image'
+
+function Box({data}){
+    return(
+        <Container>
+            <LeftSection>
+                { data?.imageUrl?.length > 0 && (<Image src={data.imageUrl} alt={data.name} width="350" height="200"/>) }
```

Comment:
Do not use height & width with the images as the image will loose the responsive ness.